Merge debian repo

arpio23 opened the pull request:
Merge from debian salsa, uses now beaconDB. The last commit also fixes the speed reporting almost double of real value.

We should test it on few devices, I tested on bronco with Firefox and graphhopper navigation.

bigon committed:
debian/watch: Fix the logic after GitLab changes

bigon committed:
New upstream version 2.7.2

bigon committed:
Update upstream source from tag 'upstream/2.7.2'

Update to upstream version '2.7.2'
with Debian dir fff2241274600c71d7c8d3ae54fffe5922323ee4

bigon committed:
New upstream release

bigon committed:
Bump build-dependencies

bigon committed:
debian/geoclue-2.0.lintian-overrides: Drop package-supports-alternative-init-but-no-init.d-script, not used anymore

bigon committed:
debian/control: Bump Standards-Version to 4.7.0 (no further changes)

bigon committed:
debian/control: Update build-dependencies to please lintian: pkg-config => pkgconf

bigon committed:
debian/rules: Use beaconDB as a source, with the maintainer approval (Closes: #1074427)

bigon committed:
debian/changelog: Add missing Closes statement

bigon committed:
debian/patches: Cherry-pick several patches for beaconDB

bigon committed:
Release to unstable

bigon committed:
debian/control: Add Provides: ${gir:Provides} for the -dev package

bigon committed:
debian/rules: Use api.beacondb.net instead of beacondb.net

aleasto committed:
Fix AppArmor rules for DBus

Verified working on Ubuntu.

I've tried securing most of the calls, however because this service registers
objects and sends signals about those it must be in some way allowed to send
messages to anyone on the bus.
For this reason, you'll find that for non-sensitive object paths I have not
restricted the peer name.

LP: #2030951

kop316 committed:
debian/rules: Make beacondb.net default for wifi submissions

bigon committed:
Release to unstable

bigon committed:
debian/README.Debian: Update the README to explicitly state we are using beaconDB

arpio23 committed:
Merge remote-tracking branch 'debian/master' into droidian

arpio23 committed:
hybris-binder: return MPS for speed.

g7 reviewed (approved):
Thanks! Looks good to me, added to 101 for wider testing

bigon committed:
Update debian/changelog

bigon committed:
debian/copyright: Update the FSF address

bigon committed:
debian/control: Bump Standards-Version to 4.7.2 (no further changes)

bigon committed:
Update debian/changelog

jbicha committed:
Fix debian/upstream/metadata

jbicha committed:
New upstream version 2.8.0

jbicha committed:
Update upstream source from tag 'upstream/2.8.0'

Update to upstream version '2.8.0'
with Debian dir 3b9ab6e1c6259d7fe879aad1a2836f3c8db34b7a

jbicha committed:
New upstream release

jbicha committed:
Remove all patches: applied in new release

jbicha committed:
Remove obsolete configure flags

These are already the default values

jbicha committed:
Update gir handling

Closes: #1118821

jbicha committed:
Simplify docs build

jbicha committed:
Remove obsolete dh_auto_test override

dh_auto_test doesn't do anything for this package anyway

jbicha committed:
Bump debhelper compat to 13

jbicha committed:
Remove obsolete maintscript

this was already done in Debian 13 and Ubuntu 22.04 LTS

jbicha committed:
Use sysusers.d instead of adduser

jbicha committed:
releasing package geoclue-2.0 version 2.8.0-1

bigon committed:
New upstream version 2.8.1

bigon committed:
Update upstream source from tag 'upstream/2.8.1'

Update to upstream version '2.8.1'
with Debian dir 172b4886fcc95e85b1fff66f0ccce966bd76c6dc

bigon committed:
New upstream release

arpio23 committed:
Merge remote-tracking branch 'debian/master' into feature/next/2.7.2
